sdk/src/Core/Internal/Utils/AppSettings.netcore.cs

```diff
@@ -63,9 +78,14 @@ public XRayOptions(string pluginSetting, string samplingRuleManifest, string aws
         /// </summary>
         public string AwsServiceHandlerManifest { get => _awsServiceHandlerManifest; set => _awsServiceHandlerManifest = value; }
 
-        /// <summary>
-        /// Tracing disabled value, either true or false.
-        /// </summary>
+        /// <summary>
+        /// Tracing disabled value, either true or false.
+        /// </summary>
         public bool IsXRayTracingDisabled { get => _isXRayTracingDisabled; set => _isXRayTracingDisabled = value; }
+
+        /// <summary>
+        /// Determines whether runtime errors should occur when recording of data is attempted and a tracing header is not available, either true or false.
```

Comment:
Hi @stevejgordon ,
Can we modify this comment to `For missing Segments/Subsegments, if set to true, runtime exception is thrown, if set to false, runtime exceptions are avoided and logged.`
Reference:  "Environment Variables" section - https://docs.aws.amazon.com/xray/latest/devguide/xray-sdk-dotnet-configuration.html